src/Text.Analyzers/Core/IdentifiersShouldBeSpelledCorrectly.cs

```diff
@@ -409,7 +435,12 @@ IEnumerable<string> GetMisspelledWords(string symbolName)
             static bool IsWordNumeric(string word) => char.IsDigit(word[0]);
 
             bool IsWordSpelledCorrectly(string word)
-                => !projectDictionary.UnrecognizedWords.Contains(word) && projectDictionary.RecognizedWords.Contains(word);
+            {
+                var unrecognizedWordsContains = dictionaries.Any(static (d, word) => d.ContainsUnrecognizedWord(word), word);
+                var recognizedWordsContains = dictionaries.Any(static (d, word) => d.ContainsRecognizedWord(word), word);
```

Comment:
>var recognizedWordsContains = dictionaries.Any(static (d, word) => d.ContainsRecognizedWord(word), word); [](http://example.com/codeflow?start=16&length=105)

nit: the new code always executes both lookups, maybe it's worth it for the additional clarity?

Reply:
no.  it would def make sense to shortcircuit.  i'll do that.